paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -117,29 +124,35 @@ internal class PaymentOptionsViewModel @Inject constructor(
         }
     }
 
-    internal sealed class TransitionTarget {
+    internal sealed class TransitionTarget : Parcelable {
```

Comment:
Check, but I don't think we save this to instance state any more.

Reply:
Does it need to be Parcelable then?

Reply:
nope, removed